code/modules/stashes/stash_datum.dm

```diff
@@ -280,6 +280,16 @@ This file contains the underlying code for stash datums
 			//Spawn it in the thing
 			results += new a(T)
 
+	// Going thru the list to hide under the floor
+	for(var/h in results)
+		// Better hide remains under the tile.
+		if (istype(h, /obj/item/remains))
+			var/obj/item/remains/R = h
```

Comment:
In this case, `for(var/obj/item/remains/R in results)` would be a better option. It basically folds istype into the loop.

Reply:
Ah, it would clear up how the dm code works.